minions/async/src/test/java/io/github/microcks/minion/async/AsyncAPITestManagerIT.java

```diff
@@ -166,4 +166,94 @@ private static void sendTextMessagesOnTopic(int number) {
       producer.flush();
       producer.close();
    }
+
+   @Test
+   void testKafkaSendMessageTestSuccess() throws Exception {
+      // Arrange.
+      String asyncAPIContent = Files.readString(
+            Paths.get("target/test-classes/io/github/microcks/minion/async", "send-message-asyncapi-3.0.yaml"));
+      Map<String, TestCaseReturnDTO> reportedTestCases = new HashMap<>();
+      MicrocksAPIConnector microcksAPIConnector = new MicrocksAPIConnector() {
+         @Override
+         public KeycloakConfig getKeycloakConfig() {
+            return null;
+         }
+
+         @Override
+         public List<Service> listServices(String authorization, int page, int size) {
+            return null;
+         }
+
+         @Override
+         public ServiceView getService(String authorization, String serviceId, boolean messages) {
+            return null;
+         }
+
+         @Override
+         public List<Resource> getResources(String serviceId) {
+            if ("d3d5a3ed-13bf-493f-a06d-bf93392f420b".equals(serviceId)) {
+               Resource resource = new Resource();
+               resource.setId("578497d2-2695-4aff-b7c6-ff7b9a373aa9");
+               resource.setType(ResourceType.ASYNC_API_SPEC);
+               resource.setContent(asyncAPIContent);
+               return List.of(resource);
+            }
+            return Collections.emptyList();
+         }
+
+         @Override
+         public TestCaseResult reportTestCaseResult(String testResultId, TestCaseReturnDTO testCaseReturn) {
+            reportedTestCases.put(testResultId, testCaseReturn);
+            return null;
+         }
+      };
```

Comment:
Maybe we could extract the anonymous implementation present in different tests to a named class to avoid duplication?